### ISSUE TITLE
Sample video?

### ISSUE THREAD

**Issue by fromEON** [Zukhruf/Animation-Engine-with-Kinect#1]
Is there any video samples for results? And any plans for Intel realsense D series cameras?

**Comment by Zukhruf** [comment]
Yes, I have video results from it, but I haven't published yet. Maybe a couple of days I will post it on youtube.

I never have known what intel realsense is? Is the technology almost the same as Kinect? I will do some research about it, and if there any chances for me to try it, I would love to.

Zukhruf
